documentation/persistent-vols-claims.md

```diff
@@ -2,9 +2,9 @@
 title: Persistent Volumes & Persistent Volumes Claims
 ---
 
-In case of Kubernetes Volumes we know that once the Pod is deleted the specification of the volume in the Pod is also lost. Even though VMDK file persists but from Kubernetes perspective the volume is deleted.
+In case of Kubernetes Volumes, we know that once the Pod is deleted the specification of the volume in the Pod is also lost. Even though VMDK file persists but from Kubernetes perspective the volume is deleted.
```

Comment:
`from the perspective of Kubernetes` or `Kubernetes's perspective`

Reply:
Fixed.